wp-settings-framework.php

```diff
@@ -985,7 +985,12 @@ public function generate_toggle_field( $args ) {
 			$checked       = ( $args['value'] ) ? 'checked="checked"' : '';
 
 			echo '<input type="hidden" name="' . esc_attr( $args['name'] ) . '" value="0" />';
-			echo '<label class="switch"><input type="checkbox" name="' . esc_attr( $args['name'] ) . '" id="' . esc_attr( $args['id'] ) . '" value="1" class="' . esc_attr( $args['class'] ) . '" ' . esc_html( $checked ) . '> ' . esc_html( $args['desc'] ) . '<span class="slider"></span></label>';
+			echo '<label class="switch">
+					<input type="checkbox" name="' . esc_attr( $args['name'] ) . '" id="' . esc_attr( $args['id'] ) . '" value="1" class="' . esc_attr( $args['class'] ) . 
+					'" ' . esc_html( $checked ) . '> ' .
```

Comment:
@pramodjodhani I suggest using the function [checked](https://developer.wordpress.org/reference/functions/checked/)